### PR TITLE
feat(Avatar): Add initials

### PR DESCRIPTION
**Pull request by ashwoodall**
This PR adds the ability for a name prop to be passed to Avatar and creates initials if an image src is not defined. It handles passed first + last name, first (single) name, initials (2 chars), or no name prop.

### CODE REVIEW

File: src/components/Avatar.js [hunk 1]
```diff
@@ -2,8 +2,22 @@ import React from 'react';
...
+  const nameArr = name.split(' ');
+  const initials = nameArr.length === 1 ? nameArr[0][0] : nameArr[0][0] + nameArr[1][0];
+
+  return initials.toUpperCase();
+};
+
 export const Avatar = Circular.extend.attrs({
-  children: props => (props.src ? <Image src={props.src} /> : props.children),
+  children: props =>
+    (props.src ? <Image src={props.src} /> : createInitials(props.name, props.children)),
```
Comment (martahj) [inline src/components/Avatar.js]: This is kind of a NIT and you don't really have to do this, but I find it somewhat confusing to have the logic for returning the children combined with the logic for creating the initials. The function of `createInitials` might be clearer if it was broken into two functions, one that created the initials and one that determined whether to return initials or children... or I guess if `createInitials` was renamed

**Comment by martahj** [comment]
This is looking good!

Since we're temporarily unable to use Now, could you please do the following:
* Pull in the latest change
* Add a working example of this being used to a demo page
* Run `yarn dev`
* Verbally verify that this is behaving as expected

File: src/components/Avatar.js (hunk 1 above)
Comment (ashwoodall) [reply src/components/Avatar.js]: For the sake of keeping the ternary clean I destructured props and renamed the function to `formatProps`. LMK if this makes more sense, and if not I don't mind breaking it out into separate functions.

**Comment by martahj** [comment]
Such a beautiful demo page. Look at that gradient behind 'hi'.

**Pull request merged**